src/writer/vault.py

```
@@ -0,0 +1,53 @@
+import os
+from typing import Dict, Union
+import logging
+import requests
+
+
+class WriterVault:
```

Pylint (3.3.7)

[convention] Missing class docstring

(C0115)